run_hmm.c

```diff
@@ -1,5 +1,6 @@
 #include "hmm.h"
 #include "run_hmm.h"
+#include "config.h"  // TODO Missing include for PKGDATADIR?
```

Comment:
@htsuji31 Just checking -- how does this additional `include` change the code? Does this fix a different bug than #2? If so, maybe we should include this code change in a separate commit and/or PR.

I also noticed that you added a TODO here -- does this represent a low-priority issue, or could it potentially cause other issues with the code? Thanks.

Reply:
I ask because it's the only code change in this file.

Reply:
Either I noticed an error compiling run_hmm.c (maybe when I was trying to manually compile the file outside the build environment used for this project) or thought this file shouldn't be able to compile because it *looked like* it had no means of including config.h (which contains the definition for PKGDATADIR, which is used in run_hmm.c) -- I don't remember which was the case here. The TODO is documenting the problem I was "fixing" by adding the #include.
If there are no build errors when you don't include this change (which is probably the case), you can just omit it from the pull request. In that case, either (1) adding this #include isn't necessary (e.g. because config.h is being included by some other .h file included by run_hmm.c), or (2) something in the build process is hiding or compensating for the fact that this #include is missing.

Reply:
@htsuji31 Thanks -- this is helpful. I'll omit this change from the PR, and it can be addressed in the future if a bug occurs during compilation.